src/ray/raylet/event/events.h

```diff
@@ -0,0 +1,302 @@
+#ifndef RAY_EVENT_EVENTS_H
+#define RAY_EVENT_EVENTS_H
+
+#include <memory>
+#include <unordered_map>
+#include <unordered_set>
+#include <vector>
+
+#include "ray/id.h"
+
+namespace ray {
+
+namespace events {
+
+using SubscriptionID = UniqueID;
```

Comment:
I've seen this in a couple places. This should be moved to `id.h` with all the other ID types.